packages/avatar/package.json

```diff
@@ -19,6 +19,7 @@
   ],
   "dependencies": {
     "classnames": "^2.2.5",
+    "emotion": "^10.0.4",
```

Comment:
i think we may want to put this at `10.0.0` as our other themeable components are currently at 10. could also remove `classnames` and `@hig/styles` as i don't think those will be used any longer.